app/backplane/radio_module/src/c_radio_module.cpp

```diff
@@ -0,0 +1,37 @@
+#include "c_radio_module.h"
+
+// F-Core Tenant
+#include <f_core/os/n_rtos.h>
+#include <f_core/messaging/c_msgq_message_port.h>
+#include <zephyr/drivers/gnss.h>
+
+K_MSGQ_DEFINE(broadcastQueue, 256, 10, 4);
+static auto loraBroadcastMsgQueue = CMsgqMessagePort<NRadioModuleTypes::RadioBroadcastData>(broadcastQueue);
+static auto udpBroadcastMsgQueue = CMsgqMessagePort<NRadioModuleTypes::RadioBroadcastData>(broadcastQueue);
+
+CRadioModule::CRadioModule() : CProjectConfiguration(), lora(*DEVICE_DT_GET(DT_ALIAS(lora))),
+                               loraBroadcastMessagePort(loraBroadcastMsgQueue), udpBroadcastMessagePort(udpBroadcastMsgQueue) {
+}
+
+void CRadioModule::AddTenantsToTasks() {
+    // Networking
+    networkingTask.AddTenant(sensorModuleListenerTenant);
```

Comment:
i think ur cooked. sensorModuleListenerTenant and powerModuleListenerTenant both get added to network task. 
But their both udp_listener_tenants whose Run method `while true` s 

so network task will start but get locked up by sensorModuleListenerTenant and never get to powermoduleListenerTenant and then infinite sadness.

Reply:
Oops will be fixed